test/visual_regression/tests/discover/chart_visualization.ts

```diff
@@ -33,8 +36,9 @@ export default function ({ getService, getPageObjects }: FtrProviderContext) {
       await PageObjects.timePicker.setDefaultAbsoluteRange();
     });
 
-    after(function unloadMakelogs() {
+    after(async function unloadMakelogs() {
       return esArchiver.unload('test/functional/fixtures/es_archiver/logstash_functional');
+      await kibanaServer.savedObjects.cleanStandardList();
```

Comment:
If line 40 has `return` then line 41 never runs?  I think return should be replaced with await.

Reply:
wow! i did not see that return.